network/p2p/throttler_test.go

```diff
@@ -0,0 +1,155 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package p2p
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/ava-labs/avalanchego/ids"
+)
+
+func TestSlidingWindowThrottlerHandle(t *testing.T) {
+	period := time.Minute
+	previousWindowStartTime := time.Time{}
```

Comment:
nit: could this be time.Now(). I like it slightly changes from test to test

Reply:
I just didn't do `time.Now()` to avoid the variance in tests. I don't think it matters in this test specifically but I just generally try to avoid using current time/rng/anything that might change from test-to-test in unit tests. I think for something like an integration test it's fine to do though.